README.md

```diff
@@ -180,12 +180,14 @@ Usage instructions for each are below.
 ```
 cmake . && make
 ```
-Note that you can also use `ninja` as your generator: `cmake -G Ninja . && ninja`
 
-* A C++11 compiler is required.
-* The JavaScript components can be built using `build-js.sh`, see notes inside. Normally this is not needed as builds are provided in this repo already.
+A C++14 compiler is required. Note that you can also use `ninja` as your generator: `cmake -G Ninja . && ninja`.
 
-If you also want to compile C/C++ to WebAssembly (and not just asm.js to WebAssembly), you'll need Emscripten. You'll need the `incoming` branch there (which you can get via [the SDK](http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html)), for more details see [the wiki](https://github.com/kripken/emscripten/wiki/WebAssembly).
+Binaryen.js can be built using Emscripten, which can be emgotten via [the SDK](http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html)). For more details see [the wiki](https://github.com/kripken/emscripten/wiki/WebAssembly).
```

Comment:
I think the second sentence can be removed - the link is old and not necessary.

:smile: on emgotten but maybe it could confuse some users.